Bento/Renderable/AnyRenderable.swift

```diff
@@ -1,14 +1,10 @@
 import UIKit
 
-struct AnyRenderable: Renderable, Deletable {
-    var canBeDeleted: Bool {
-        return base.canBeDeleted
-    }
+protocol ExistentialWrapper {
 
-    var deleteActionText: String {
-        return base.deleteActionText
-    }
+}
```

Comment:
[nit] extra line above. Actually since this is a marker protocol, could just be `{ }`